github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -42,7 +42,7 @@ presubmits:
     annotations:
       fork-per-release: "true"
       testgrid-dashboards: kubevirt-presubmits
-    always_run: false
+    always_run: true
```

Let's keep the `pull-kubevirt-e2e-k8s-1.20-cgroupsv2` lane executed on-demand.
```suggestion
    always_run: false
```